inc/namespace.php

```diff
@@ -53,17 +58,21 @@ function enqueue_assets() {
 		$ver .= '-' . filemtime( plugin_dir_path( __DIR__ ) . 'dist/css/styles.css' );
 	}
 
-	wp_enqueue_script( 'altis-consent', $js, [], $ver, true );
+	wp_enqueue_script( 'altis-consent', $js, [ 'altis-consent-api' ], $ver, true );
 	wp_enqueue_style( 'altis-consent', $css, [], $ver, 'screen' );
```

Comment:
I think we can not enqueue this if the banner isn't going to be shown / loaded. Assuming I understand the intention of that setting. If the markup isn't there then we can save a few bytes and an HTTP request by not loading this.

Reply:
You mean by checking if the option to display the banner is turned off?

Reply:
Yeah precisely that 👍